functions/__bobthefish_glyphs.fish

```diff
@@ -30,6 +30,9 @@ function __bobthefish_glyphs -S -d 'Define glyphs used by bobthefish'
   # Kubernetes glyphs
   set -x k8s_glyph               \u2388 # '⎈'
 
+  # AWS glyphs
+  set -x aws_vault_glyph         \u2601 # '☁'
```

Comment:
i … don't love this cloud. it took me a long time to figure out what it even was.

we don't _have_ to have a glyph for everything. if there's not a great fit, it's not the end of the world to leave it off, or to use `AWS` or something.

Reply:
Happy to leave it off as well. As you can see in 398a60b I wasn't even convinced.